onnx/version_converter/convert.h

```diff
@@ -563,9 +563,17 @@ class DefaultVersionConverter : public BaseVersionConverter {
     registerAdapter(std::make_unique<CompatibleAdapter>("Shape", OpSetID(18), OpSetID(19)));
     registerAdapter(std::make_unique<CompatibleAdapter>("Size", OpSetID(18), OpSetID(19)));
 
+    /******** 18 -> 20 ********/
+    registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMax", OpSetID(18), OpSetID(20)));
+    registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMin", OpSetID(18), OpSetID(20)));
```

Comment:
```suggestion
    /******** 19 -> 20 ********/
    registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMax", OpSetID(19), OpSetID(20)));
    registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMin", OpSetID(19), OpSetID(20)));
```
You should just use the previous version instead of the last available version for that op.

Reply:
Thanks you! Curious why this is?

Reply:
I don't have the history context here, but I guess it's just more convenient for design purpose? Version converter actually converts the version one by one and it's easier to detect whether there's a required conversion.